src/pages/preferences/preferencesTabs/cliTab/ConfirmationCard.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
```

Comment:
nit: This file name gave me the impression that the user is meant to interact and confirm something. I think it makes more sense to name this VerifyCard